README.md

```diff
@@ -0,0 +1,93 @@
+# widelands-flatpak
```

Comment:
Half of this file is pointless duplication of official docs. It's fine to keep really short instructions how to build it if you feel an urge, but it's too much now in my opinion.